webapp/app/config/services.yml

```diff
@@ -67,3 +67,6 @@ services:
     DOMJudgeBundle\Controller\Jury\ImportExportController:
         arguments:
             $domjudgeVersion: '%domjudge.version%'
+
+    DOMJudgeBundle\Security\SimpleSAMLAuthenticator:
+        arguments: ["@router", "@ssp.guard.registry", "surfconext"]
```

Comment:
This last argument should be a parameter I guess, so you can configure it. Or if we move it to the DB anyway, read it from the DB.

Reply:
This parameter is probably not right anyway since the login page allows to select which authsource to use.

Reply:
I have overriden this requirement now. Asked upstream about the possibilities to fix it there in pr 6